pkg/api/client/amqp/qeclients/factoryreceiver.go

```diff
@@ -38,6 +38,11 @@ func (a *AmqpQEReceiverBuilder) WithCount(count int) *AmqpQEReceiverBuilder {
 	return a
 }
 
+func (a *AmqpQEReceiverBuilder) WithCustomCommand(command string) *AmqpQEReceiverBuilder {
+	a.customCommand = command
+	return a
+}
+
```

Comment:
Why not adding this to: factory.go, as part of AmqpQEClientBuilderCommon?

Reply:
Oh, true that.